### PR TITLE
Dedup + CLI instead of ENV

### PR DESCRIPTION
**Pull request by coreagile**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ README.pdf
 pom.xml
 pom.xml.asc
 profiles.clj
+.idea
+oatmeal.iml
```
Comment (eigenhombre) [inline .gitignore]:
```suggestion
oatmeal.iml

```

**Review by eigenhombre — COMMENTED** [review]
I think I don't mind this approach overall, but it looks like the unit tests fail.  Can you `lein test` locally to verify?  If I get a minute I can try to look more deeply into the failure myself.

File: .gitignore (hunk 1 above)
Comment (coreagile) [reply .gitignore]: > I think I don't mind this approach overall, but it looks like the unit tests fail. Can you `lein test` locally to verify? If I get a minute I can try to look more deeply into the failure myself.

Tests fixed!

**Comment by eigenhombre** [comment]
@coreagile I did merge this but it broke the build; I notice the build is also broken in your fork.  I reverted the merge to keep the build green.  Haven't had a chance to figure out why the build is still failing for you: https://github.com/coreagile/oatmeal/runs/7216008644

**Comment by coreagile** [comment]
Very strange that it worked on my machine lol.

Stephen Castro-Starkey
empathy | connectedness | restorative | relator | belief

secure email: https://keybase.io/scstarkey <https://keybase.io/scstarkey> 

Gallup Certified Strengths Coach

**Comment by coreagile** [comment]
Looks like it’s only breaking in Docker. Gotta figure that out I suppose!

Stephen Castro-Starkey
empathy | connectedness | restorative | relator | belief

secure email: https://keybase.io/scstarkey <https://keybase.io/scstarkey> 

Gallup Certified Strengths Coach

**Pull request merged**